test/uk/gov/hmrc/thirdpartyorchestrator/controllers/ApplicationControllerSpec.scala

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.thirdpartyorchestrator.controllers
+
+import scala.concurrent.ExecutionContext.Implicits.global
+
+import org.scalatest.matchers.should.Matchers
+
+import play.api.http.Status
+import play.api.test.Helpers._
+import play.api.test.{FakeRequest, Helpers}
+import uk.gov.hmrc.http.HeaderCarrier
+
+import uk.gov.hmrc.apiplatform.modules.common.domain.models._
+import uk.gov.hmrc.thirdpartyorchestrator.mocks.services.ApplicationServiceMock
+import uk.gov.hmrc.thirdpartyorchestrator.utils.DeveloperBuilder
+
+class ApplicationControllerSpec extends BaseControllerSpec with Matchers {
+
+  trait Setup
+      extends ApplicationServiceMock with DeveloperBuilder {
+
+    implicit val hc = HeaderCarrier()
+
+    val applicationId = ApplicationId.random
+    val userId        = UserId.random
+    val email         = LaxEmailAddress("bob@example.com")
+    val developer     = buildMinimalDeveloper(userId, email, "Bob", "Fleming")
+    val request       = FakeRequest()
+    val controller    = new ApplicationController(applicationServiceMock, Helpers.stubControllerComponents())
+  }
+
+  "getVerifiedCollaboratorsForApplication" should {
+    "return 200 if successful" in new Setup {
+      fetchVerifiedCollaboratorsForApplicationReturns(applicationId, Set(developer))
+      val result = controller.getVerifiedCollaboratorsForApplication(applicationId)(request)
+      status(result) shouldBe Status.OK
+    }
+
+    "return 200 if appplication found but no verified developers" in new Setup {
+      fetchVerifiedCollaboratorsForApplicationReturnsNone(applicationId)
+      val result = controller.getVerifiedCollaboratorsForApplication(applicationId)(request)
+      status(result) shouldBe Status.OK
+    }
+
```

Comment:
Could we test for applicationId is not   UUID?

Reply:
Unit test - won't compile

Reply:
Can it be done in an integration test?

Reply:
Discussed with Pete. Small reward on adding an integration test just for that. It will only be used by our other services.